test/integration/react-browser/package.json

```diff
@@ -2,10 +2,13 @@
   "private": true,
   "description": "chart.js should work in react-browser (Web)",
   "dependencies": {
-    "chart.js": "file:../package.tgz",
+    "@babel/core": "^7.0.0",
+    "@babel/plugin-syntax-flow": "^7.14.5",
+    "@babel/plugin-transform-react-jsx": "^7.14.9",
+    "chart.js": "workspace:*",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-scripts": "5.0.1",
```

Comment:
Won't this break the test since the test relies on `react-scripts` to build the app, also it does not use TS  so why is that necesarry for this sub project

Reply:
oops, yes. I accidentally removed that while figuring something out. I've added back!

`typescript` is there because `react-scripts` declares it as a required peer dependency: https://github.com/facebook/create-react-app/blob/f34d88e30c7d8be7181f728d1abc4fd8d5cd07d3/packages/react-scripts/package.json#L88